roles/openshift_project_request_template/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+- when: openshift_project_request_template_manage
+  block:
```

Comment:
No need for block if bool is moved to calling the role.

Reply:
Moved the condition to where the role is called.